src/dev.sh

```diff
@@ -16,6 +16,7 @@ PACKAGE_DIR="$SCRIPT_DIR/../_package"
 DOTNETSDK_ROOT="$SCRIPT_DIR/../_dotnetsdk"
 DOTNETSDK_VERSION="2.1.403"
 DOTNETSDK_INSTALLDIR="$DOTNETSDK_ROOT/$DOTNETSDK_VERSION"
+AGENT_VERSION="2.147.0"
```

Comment:
I would rather have the version in a agentversion file with this script doing something like

```
AGENT_VERSION=`cat agentversion`
```

Reply:
Sure thing! I pushed a new change.